pom.xml

```diff
@@ -8,6 +8,7 @@
   <packaging>pom</packaging>
   <version>1.0.0</version><!-- Need not change for every release-->
   <modules>
+    <module>eng/azure-test-watcher</module>
```

Comment:
Is there a reason this can't be lumped into azure-core-test? It's only used for tests. It'll give azure-core-test a reason to live.

Reply:
Won't be usable in Azure Core due to circular dependencies.

Reply:
@alzimmermsft this should probably be in the common with perf-test-core and smoke-tests, instead of eng.

Reply:
Moving this into `common` makes sense to me, will move it in the next commit.

Reply:
Moved into common